### PR TITLE
removing unused events

### PR DESCRIPTION
**Pull request by dana-yaish**
### Purpose/Motivation
Removing non used events from the code. You can access the whole investigation in [here](https://www.notion.so/sentry/Codecov-Segment-deprecation-handling-events-62cb4c0e1d2f41cf865140d676aa02f6) 

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. In 2022 this entity acquired Codecov and as result Sentry is going to need some rights from me in order to utilize my contributions in this PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: plan/service.py [hunk 1]
```diff
@@ -124,15 +124,6 @@ def start_trial(self) -> None:
         self.current_org.plan_auto_activate = True
         self.current_org.save()
 
-        self.notifier_service.trial_started(
```
Comment (giovanni-guidini) [inline plan/service.py:127]: Maybe you can also delete `self.notifier_service` in line 26. It's not being used anymore, apparently

File: webhook_handlers/views/github.py [hunk 2]
```diff
@@ -408,13 +408,6 @@ def _handle_installation_events(self, request, *args, **kwargs):
                 "Owner deleted app integration",
                 extra=dict(ownerid=owner.ownerid, github_webhook_event=self.event),
             )
-            self.segment_service.account_uninstalled_source_control_service_app(
```
Comment (giovanni-guidini) [inline webhook_handlers/views/github.py:411]: Same as above, maybe delete `self.segment_service` (line 62)

**Review by giovanni-guidini — APPROVED** [review]
Looks good in general. Lots of code removed.
Left 2 comments, but it's ok

**Comment by dana-yaish** [comment]
## [Codecov](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov) Report
> Merging [#116](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov) (c1dac00) into [main](https://app.codecov.io/gh/codecov/codecov-api/commit/9b23dd336ee984f48f78817bb9de516f501accb7?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov) (9b23dd3) will **decrease** coverage by `0.63%`.
> Report is 1 commits behind head on main.
> The diff coverage is `83.33%`.

:exclamation: Your organization needs to install the [Codecov GitHub app](https://github.com/apps/codecov/installations/select_target) to enable full functionality.

[Impacted file tree graph](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov)

```diff
@@            Coverage Diff             @@
##             main     #116      +/-   ##
==========================================
- Coverage   86.32%   85.69%   -0.63%     
==========================================
  Files         577      576       -1     
  Lines       16213    16290      +77     
==========================================
- Hits        13996    13960      -36     
- Misses       2217     2330     +113     
```


| [Files](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov) | Coverage Δ | |
|---|---|---|
| [upload/views/legacy.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-dXBsb2FkL3ZpZXdzL2xlZ2FjeS5weQ==) | `80.52% <77.77%> (ø)` | |
| [api/internal/repo/serializers.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-YXBpL2ludGVybmFsL3JlcG8vc2VyaWFsaXplcnMucHk=) | `94.73% <66.66%> (-5.27%)` | :arrow_down: |
| [api/internal/owner/serializers.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-YXBpL2ludGVybmFsL293bmVyL3NlcmlhbGl6ZXJzLnB5) | `88.28% <ø> (+6.87%)` | :arrow_up: |
| [api/internal/owner/views.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-YXBpL2ludGVybmFsL293bmVyL3ZpZXdzLnB5) | `79.67% <ø> (-18.13%)` | :arrow_down: |
| [api/internal/repo/views.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-YXBpL2ludGVybmFsL3JlcG8vdmlld3MucHk=) | `80.53% <ø> (-19.47%)` | :arrow_down: |
| [billing/views.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-YmlsbGluZy92aWV3cy5weQ==) | `83.47% <ø> (-4.37%)` | :arrow_down: |
| [codecov\_auth/views/base.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-Y29kZWNvdl9hdXRoL3ZpZXdzL2Jhc2UucHk=) | `96.20% <100.00%> (-3.30%)` | :arrow_down: |
| [plan/service.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-cGxhbi9zZXJ2aWNlLnB5) | `66.14% <ø> (+6.14%)` | :arrow_up: |
| [profiling/views.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-cHJvZmlsaW5nL3ZpZXdzLnB5) | `82.69% <ø> (-17.31%)` | :arrow_down: |
| [services/activation.py](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov#diff-c2VydmljZXMvYWN0aXZhdGlvbi5weQ==) | `69.13% <ø> (-23.97%)` | :arrow_down: |
| ... and [3 more](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree-more&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov) | |

[Impacted file tree graph](https://app.codecov.io/gh/codecov/codecov-api/pull/116?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=codecov)

**Pull request merged**